launch/driver.launch.py

```diff
@@ -7,7 +7,7 @@
 
 from launch import LaunchDescription, conditions
 from launch.actions import (DeclareLaunchArgument, GroupAction)
-from launch.substitutions import LaunchConfiguration
+from launch.substitutions import LaunchConfiguration, Command
```

Comment:
is this change needed?

Reply:
sorry should have commented this. It's not my change, it's just a revert of our changes to match the original repo. It makes no difference at all, just wanted to have an easier picture when comparing our fork

Reply:
thanks!